From 7d984530888aa32c5b3cb9a5d4a9e60eab48adc2 Mon Sep 17 00:00:00 2001
From: Elliott Hughes <enh@google.com>
Date: Tue, 2 Dec 2014 16:59:38 -0800
Subject: [PATCH 82/95] __ARM_HAVE_HALFWORD_MULTIPLY is always true for
 __arm__.

And this is the only user of that macro...

Bug: 18556103
Change-Id: I1b515a54ac8530b150f4dd93070db60c701a5603
---
 services/audioflinger/AudioResampler.cpp | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/services/audioflinger/AudioResampler.cpp b/services/audioflinger/AudioResampler.cpp
index 1f7a613..46e3d6c 100644
--- a/services/audioflinger/AudioResampler.cpp
+++ b/services/audioflinger/AudioResampler.cpp
@@ -29,14 +29,11 @@
 #include "AudioResamplerDyn.h"
 
 #ifdef __arm__
-#include <machine/cpu-features.h>
+    #define ASM_ARM_RESAMP1 // enable asm optimisation for ResamplerOrder1
 #endif
 
 namespace android {
 
-#ifdef __ARM_HAVE_HALFWORD_MULTIPLY // optimized asm option
-    #define ASM_ARM_RESAMP1 // enable asm optimisation for ResamplerOrder1
-#endif // __ARM_HAVE_HALFWORD_MULTIPLY
 // ----------------------------------------------------------------------------
 
 class AudioResamplerOrder1 : public AudioResampler {
-- 
2.5.0

